docs/container-images.md

```diff
@@ -117,6 +117,9 @@ Options:
                         the pipeline, using the configured build plugin.
                         Defaults to `False`. If not set, the pipeline will be
                         run locally in the current environment.
+  -c, --clean           When `--build` is specified, builds the image from
```

Comment:
Docker calls this option [`--no-cache`](https://docs.docker.com/engine/reference/commandline/build/), we might want to do the same to prime people's brains with appropriate preconceptions. Also, since we already have a `sematic clean` option, `--no-cache` would reduce the ambiguity.

Reply:
Done.